plugins/BEdita/Core/src/ORM/Locator/TableLocator.php

```diff
@@ -52,12 +59,14 @@ protected function _getClassName($alias, array $options = [])
             return $className;
         }
 
-        try {
-            $objectTypes = $this->get('ObjectTypes');
-            $objectType = $objectTypes->get($alias);
-            $options['className'] = $objectType->table;
-        } catch (\Exception $e) {
-            $this->log($e->getMessage(), 'warning');
+        if (!in_array($alias, $this->excludeAlias)) {
```

Comment:
It is already impossible to have an object type whose name starts with an underscore, since object type aliases are forcibly camelized through inflector.